.github/workflows/deploy-func.yaml

```
@@ -35,77 +34,78 @@ on:
         default: us-central1
         type: string
       func-data-local-path:
-        default: 'data'
+        default: data
         type: string
-        description: 'Path under func-src-dir where the GCS archive is extracted, default ./data'
+        description: Path under func-src-dir where the GCS archive is extracted, default ./data
     secrets:
       JSON_GCLOUD_SERVICE_ACCOUNT_JSON:
         required: true
+
 jobs:
   deploy-cloud-func-helper:
-    runs-on: 'ubuntu-latest'
+    runs-on: ubuntu-latest
     env:
-      DATA_ARCHIVE_HASH: 'none'
+      DATA_ARCHIVE_HASH: none
 
     steps:
-      - uses: 'actions/checkout@v4'
+      - name: Checkout repo
+        uses: actions/checkout@v4
 
       - name: Authenticate to GCP
-        uses: 'google-github-actions/auth@v2'
+        uses: google-github-actions/auth@v2
         with:
           # workload_identity_provider: 'projects/123456789/locations/global/workloadIdentityPools/my-pool/providers/my-provider'
           # service_account: 'cloud-function-deployer@gap-som-dbmi-geneplx-app-p0n.iam.gserviceaccount.com'
           credentials_json: ${{ secrets.JSON_GCLOUD_SERVICE_ACCOUNT_JSON }}
 
       - name: Set up Cloud SDK
-        uses: 'google-github-actions/setup-gcloud@v2'
+        uses: google-github-actions/setup-gcloud@v2
         with:
-          version: '>= 363.0.0'
+          version: ">= 363.0.0"
 
-      - name: Get hash of the data archive for this function
+      - name: Get hash of data archive for func
         run: |
           gsutil ls -L ${{ inputs.func-data-gcs-url }} | \
             grep "Hash (crc32c)" | \
             awk '{printf "DATA_ARCHIVE_HASH=%s",$3}' >> "$GITHUB_ENV"
 
-      # if the cached files aren't present, this will defer caching the files
-      # until the end of a successful run of this workflow. the next time the
-      # workflow runs, it'll retrieve the cached files from the previous run
-      # and skip the download step.
-      # the hash of the current datafile is included as part of the key, so
-      # that fresh data will be fetched if the datafile has changed in GCS.
+      # if cached files aren't present, defer caching files until end of
+      # successful run. next time workflow runs, it'll retrieve cached files
+      # from previous run and skip download step. hash of current datafile
+      # included as part of key, so fresh data will be fetched if datafile has # changed in GCS.
+
```

I guess the original was too verbose? Asking so I know what writing style you prefer.

Also, seems there was a missing newline before a `#`:
```suggestion
      # if cached files aren't present, defer caching files until end of
      # successful run. next time workflow runs, it'll retrieve cached files
      # from previous run and skip download step. hash of current datafile
      # included as part of key, so fresh data will be fetched if datafile has
      # changed in GCS.
```

---

I did shorten it while i was reformatting things but it's completely inconsequential and subjective so I'd be happy to revert it.